homeassistant/components/modbus/modbus.py

```diff
@@ -158,12 +158,10 @@ async def async_modbus_setup(
 
     async def async_stop_modbus(event: Event) -> None:
         """Stop Modbus service."""
-
-        async_dispatcher_send(hass, SIGNAL_STOP_ENTITY)
```

Comment:
Why do we remove this one?

Reply:
The thought was the entity is already stopped because HA is in close state, didn't check yet if this is the case or not.

Reply:
As confirmed on Discord this is actually the case so marking this as resolved